src/libOpenImageIO/CMakeLists.txt

```diff
@@ -307,3 +307,34 @@ if (OIIO_BUILD_TESTS)
     #add_test (imagespeed_test imagespeed_test)
 
 endif (OIIO_BUILD_TESTS)
+
+# Package config support
+
+# Configure and install pkg_config file.
+if (OIIO_INSTALL_PKGCONFIG)
+  # Make sure all libraries ends with "_LIBRARIES"
+  set(WEBP_LIBRARIES ${WEBP_LIBRARY})
+  set(LIBRAW_LIBRARIES ${LibRaw_r_LIBRARIES})
+  foreach(lib IN ITEMS Boost PNG TIFF JPEG ZLIB FIELD3D OPENJPEG WEBP OpenCV OPENSSL FREETYPE GIF LIBRAW OCIO)
+    if (${lib}_FOUND)
+      list (APPEND PKG_LIBRARY_FLAGS ${${lib}_LIBRARIES})
+    endif()
+  endforeach()
+  list (APPEND PKG_LIBRARY_FLAGS ${CMAKE_DL_LIBS})
+  string(REPLACE ";" " -l" PKG_LIBRARIES_STRING "${PKG_LIBRARY_FLAGS}")
+  # Write the file.
+  file ( WRITE ${CMAKE_BINARY_DIR}/OpenImageIO.pc "prefix=${CMAKE_INSTALL_PREFIX}
+exec_prefix=\${prefix}
+libdir=\${exec_prefix}/lib
```

Comment:
Hardcoding 'lib' is a bad idea and would have to be patched by almost all distributions.
Use libdir=${CMAKE_INSTALL_LIBDIR} directly instead.